pages/common/rg.md

```diff
@@ -0,0 +1,19 @@
+# ripgrep
+
+> A fast commandline search tool.
+
+- Recursively search the current directory for a regex pattern:
+
+`rg {{pattern}}`
+
+- Search in all .gitignored and hidden files:
```

Comment:
I'd rather write this as "Include .gitignore and hidden files in the search:", or "Search for a pattern including in .gitignore and hidden files:".